cedar-testing/src/cedar_test_impl.rs

```diff
@@ -118,6 +177,26 @@ pub trait CedarTestImplementation {
         expected: Option<Value>,
     ) -> TestResult<bool>;
 
+    /// Custom evaluator entry point. The bool return value indicates the whether
+    /// evaluating the provided expression produces the expected value.
+    /// `expected` is optional to allow for the case where no return value is
+    /// expected due to errors.
```

Comment:
Added comment to the wrong function 😄

Reply:
See latest commit message for an explanation :D